tests/regression/dumptest-examples/example007-expected.csg

```diff
@@ -1,7 +1,7 @@
 multmatrix([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, -10], [0, 0, 0, 1]]) {
 	group() {
 		difference() {
-			rotate_extrude(angle = 360, convexity = 3, $fn = 0, $fa = 12, $fs = 2) {
+			rotate_extrude(convexity = 3, $fn = 0, $fa = 12, $fs = 2) {
```

Comment:
I think this needs updating (ctest -C Examples). Might as well run ctest -C All

Reply:
Yeah, oops.  When I did the initial work (where I dropped angle= from the CSG output), I fixed the tests by grepping for rotate_extrude.  When I added start=, I fixed the tests by running them and fixing them... which didn't include the examples.  Running -C All now.